pkgs/applications/networking/instant-messengers/discord/linux.nix

```diff
@@ -80,6 +80,7 @@ stdenv.mkDerivation rec {
 
     wrapProgram $out/opt/${binaryName}/${binaryName} \
         "''${gappsWrapperArgs[@]}" \
+        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--enable-features=UseOzonePlatform --ozone-platform=wayland}}" \
```

Comment:
`gappsWrapperArgs` exists, might be better to use that?

Reply:
Unfortunately `wrapGAppsHook` is AFAIK (and I lack knowledge in that area) only intended for GNOME applications (does a lot more). But I think it's often also used for GTK applications in general so that's probably why it's used for some Electron apps. It'd probably be a good idea to investigate if it's usage is appropriate for Chromium/Electron or not and use a consistent approach - but that deserves a dedicated issue.

Anyway, the implementation, for reference:

https://github.com/NixOS/nixpkgs/blob/55ec2d9e58c9975a427633494fc471111a9b41c6/pkgs/build-support/setup-hooks/wrap-gapps-hook/wrap-gapps-hook.sh#L46